pkg/cli/common/global.go

```diff
@@ -19,6 +19,10 @@ type GlobalOptions struct {
 	Timeout int
 	// Verbose mode prints out additional information
 	Verbose bool
+	// CurrentProject says which project to use if "project" flag is not passed
+	CurrentProject string
+	// CurrentCodeset sets which codeset to use if the name is not provided
+	CurrentCodeset string
```

Comment:
This is related to the crazy idea I proposed in an earlier comment: try naming these `Project` and `Codeset`, then linking them to the individual flags where they're needed in the sub-commands. This way, these variables don't only play the role of "current" project and codeset, but they also hold the value if explicitly supplied as a command like argument.